src/orion/benchmark/task/profet/conftest.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+from .profet_task import MetaModelTrainingConfig
+
+@pytest.fixture(scope="session")
```

Comment:
We should limit usage of pytest fixture to code meant for testing.

Reply:
Oh, that's a conftest file. Ok then these tests should be moved to `tests/unittests/benchmark`.